composables/layout.ts

```diff
@@ -0,0 +1,31 @@
+export const useSmallScreen = () => useState<boolean | undefined>(
+  'small-screen',
+  () => false
+)
+
+export const useNavDrawerOpen = () => useState<boolean | undefined>(
+  'nav-drawer-open',
+  () => true
+)
+
+const smallScreenWidth = 800
+
+const smallScreen = useSmallScreen()
+const navDrawerOpen = useNavDrawerOpen()
+
+if (window.innerWidth < smallScreenWidth) {
+  smallScreen.value = true
+  navDrawerOpen.value = false
+}
+
+window.addEventListener('resize', () => {
```

Comment:
I don't think we need a js window event listener to handle resize.  Vuetify comes with some built in breakpoints and also a resize directive https://vuetifyjs.com/en/directives/resize/

Reply:
I had to override that out of the box resizing / overlay scrim to reconcile the functionality of when the drawer should be open and when it shouldn't. When you see it working it'll make sense. And so I wanted to account for manually adjusting the size of a window too.

Reply:
I think a detail worth mentioning here is we want to have the nav menu open constantly on big screens and when it's small screen it can either be open or closed